htdocs/expensereport/card.php

```diff
@@ -1953,7 +1953,7 @@
 				        $totalpaid += $objp->amount;
 				        $i++;
 				    }
-				    $totalpaid = price2num($totalpaid);		// Round $totalpaid to fix floating problem after addition into loop
+				    $totalpaid = price(price2num($totalpaid));		// Round $totalpaid to fix floating problem after addition into loop
```

Comment:
price is a function to format a value so must not be used to store data in memory but when used to output data on html screen.